src/p2p.cpp

```diff
@@ -240,22 +241,22 @@ void p2p::handle_running(const code& ec, result_handler handler)
 
 session_seed::ptr p2p::attach_seed_session()
 {
-    return attach<session_seed>();
+    return attach<session_seed>(bitcoin_settings_);
```

Comment:
It should be possible to move the new settings parameter into the attach template.

Reply:
Done.